index.bs

```diff
@@ -4721,9 +4721,9 @@ Registry Definitions</h4>
 <h4 id=reg-pub>
 Publishing Registries</h4>
 
-	[=Registries=] can be published either
+	[=Registries=] can be published by [=chartered group|chartered=] or [=elected groups=] either
```

Comment:
I think we can just delete the "by" phrase, it's not adding anything.

Reply:
To me, this seems like the place in the Process which establishes who can publish a Registry. The alternative is line 3018, but that feels more like a reference to something established elsewhere than the place that establishes it.

Reply:
@fantasai , are you OK with closing this for the reason given above?